common_spree_dependencies.rb

```diff
@@ -36,7 +36,7 @@
 end
 
 group :test, :development do
-  gem 'rubocop', '~> 0.60.0', require: false
+  gem 'rubocop', '~> 0.77.0', require: false # bumped
```

Comment:
```suggestion
  gem 'rubocop', '~> 0.76.0', require: false # bumped
```

We need to keep it at 0.76 as this is the last version supported by CodeClimatee - https://docs.codeclimate.com/docs/rubocop#section-using-rubocop-s-newer-versions

Also, we need to change the code climate to actually use this version here: https://github.com/spree/spree/blob/master/.codeclimate.yml#L7

Reply:
@damianlegawiec I have updated the version, would you please check it ?